packages/x-components/static-docs/build-search-ui/web-archetype-integration-guide.md

```diff
@@ -164,30 +149,21 @@ invoked at any time. Note that you should only call this function **once**.
-| isSpa                   | `boolean`                                                                            |          | True when the X Components archetype is being run on top of an SPA.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
-| &lt;extra parameters>   | `any`                                                                                |          | Any other parameter to be sent directly to the API calls. For example, some times is needed to filter the search catalog with a warehouse parameter. In that case you can just add `warehouse: <your-warehouse-identifier>` to the snippet config.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+Check out the [X&nbsp;API](#x-api) section to learn more about the functions and parameters supported.
+
+:::
 
-### Callbacks
+::: develop Initialization using callbacks
```

Comment:
You removed the Callbacks section, so now is part of `2. Initialize Interface X` inside `Initializing Interface X project on demand`, but this is not correct, because you can set the callbacks also in `Initializing Interface X project automatically``

I liked more the way it was before, but if you want to separate it, we should have a menu like:

```
## Further considerations

### Snippet Configuration
### Setting callbacks up
### Events
### X API public functions
````

Also the paragraph at the beggining of the current Further Considerations with the three bullets is unnecesary because you are about to read each section. I feel there ir more links and info, but less clear now in this part.

Reply:
FURTHER CONSIDERATIONS neither seems the best title here to me.